pom.xml

```diff
@@ -114,7 +114,7 @@
             <groupId>*</groupId>
             <artifactId>*</artifactId>
         </exclusion>
-    </exclusions>
+      </exclusions>
```

Comment:
pls remove that change

Reply:
@npeltier  unable to see the ident or trailing space problem on my local

Reply:
right, me neither, let's move on